firmware/quadruna/CRIT/src/app/app_heartbeatMonitor.c

```diff
@@ -0,0 +1,84 @@
+#include "app_heartbeatMonitor.h"
```

Comment:
Same notes from BMS apply here - I think we can move any remaining logic in this file and move it to the heartbeat monitor library, making this just a configuration file.